### PR TITLE
Change Navigation Font Weight

### LINKED ISSUE #5665: Regression - navigation items font weight should be medium
# Current behavior

We have a regression on the navbar menu item. They are back to `font-weight: regular`

image

# Desired Behavior

The Navbar menu item font weight should be  `medium`

image

# Figma

https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=5910-77814&t=P12gjv5ZcZ9hChR8-11

### PR DESCRIPTION
**Pull request by its-id**
fixes [#5665](https://github.com/twentyhq/twenty/issues/5665).

| Before | After |
| --- | --- |
| Screenshot 2024-06-02 at 1 22 09 AM | Screenshot 2024-06-02 at 1 21 01 AM |

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const StyledItem = styled('div', {
 `;
 
 const StyledItemLabel = styled.div`
-  font-weight: ${({ theme }) => theme.font.weight.medium};
+  font-weight: ${({ theme }) => theme.font.weight.mediumForNavigation};
```
Comment (martmull) [inline packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx:87]: You can remove this change (the issue is that font.weight.medium definition is wrong)

File: packages/twenty-ui/src/theme/constants/FontCommon.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export const FONT_COMMON = {
   },
   weight: {
     regular: 400,
+    mediumForNavigation: 500,
```
Comment (martmull) [inline packages/twenty-ui/src/theme/constants/FontCommon.ts:13]: remove `mediumForNavigation`

File: packages/twenty-ui/src/theme/constants/FontCommon.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export const FONT_COMMON = {
...
     medium: 'var(--twentycrm-border-radius-sm)',
```
Comment (martmull) [inline packages/twenty-ui/src/theme/constants/FontCommon.ts:14]:
```suggestion
    medium: 'var(--twentycrm-font-weight-medium)',
```

**Review by martmull — CHANGES_REQUESTED** [review]
Hey, think you should just fix the `theme.font.weight.medium` definition instead of adding a new `mediumForNavigation` weight

File: packages/twenty-ui/src/theme/constants/FontCommon.ts (hunk 3 above)
Comment (its-id) [reply packages/twenty-ui/src/theme/constants/FontCommon.ts]: Thanks for this suggestion. I thought this was intentional and thus didn't modify the existing code.

Did the correction as asked. Please check now.

**Review by martmull — APPROVED** [review]
LGTM thanks

**Pull request merged**